index.html

```diff
@@ -0,0 +1,143 @@
+                <a class="menuOptionsLinks sponsorDesk" href="#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks" href="#"><div>News</div></a>
+                <a class="menuOptionsLinks" href="#"><div>CC Campaign</div></a>
+            </div>
+        </div>
+    </header>
+    <main>
+                                <!-- Main Section -->
+        <section class="main">
+            <div id="mobileTopContainer">
+                <nav class="hamburgerMenu">
+                    <span class="bar bar1"></span>
+                    <span class="bar bar2"></span>
+                    <span class="bar bar3"></span>
+                </nav>
+                <div id="firstMain"></div>
+            </div>
+            <!-- ---- -->
+                <img class="adornIcon taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+                <img class="adornIcon hat" src="./images/iconsMxImages/2hat-removebg-preview.png" alt="">           
+                <img class="adornIcon pinata" src="./images/iconsMxImages/3pin-removebg-preview.png" alt="">
+                <img class="adornIcon guacamole" src="./images/iconsMxImages/7guacamole-removebg-preview.png" alt="">
+            <!-- --- -->
+            <div id="textMain">
+                <div id="helloAndEventContainer">
+                    <div id="hello">"Hello! Food Lovers"</div>
+                    <h1 id="event">INTERNATIONAL MEXICAN FOOD CONVENTION 2023</h1>
+                </div>
+                <div id="description">
+                    <p id="descriptionParagraph">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries     is taking place in October, in Korea.</p>
+                </div>
+                <div id="dateAndPlaceContainer">
+                    <div id="date">2023.10.15(THU) ~ 16(FRI)</div>
+                    <div id="place">@ National Museum of Karea, Art Center Nabi and more</div>
+                </div>
+            </div>
+        </section>
+                                <!-- Program Details Section -->
+        <section id="programDetails">
+            <div class="titleAndUnderlineContainer">
+                <h2 class="title">Main Program</h2>
+                <span class="underline"></span>
+            </div>
+            <div id="programListContainer">
+                <div class="programItems">
+                    <img class="programImages" src="./images/1programMusic.png" alt="">
+                    <h3 class="programNames">Music</h3>
+                    <p class="programParagraphs">Listen to the speakers from various countries about the messages of sharing and opening.</p>
+                </div>
+                <div class="programItems">
+                    <img class="programImages" src="./images/2programComedy.png" alt="">
+                    <h3 class="programNames">Comedy Show</h3>
```

Comment:
- [ ] Notice how on the design guideline, the left and right margin of the whole content is equal and aligned. See the image below as a guide, notice the red line vertically straight which shows the boundaries. Please fix this


![Uploading image.png…]()

Reply:
Hi @bobb-Rob I don't see your image, it appears like in the image below, can you send it again?
![bobRob](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/134537383/7d4e7864-a00a-44c1-9a1d-ed6943e40386)

Reply:
I see everything aligned as shown in this video, please let me continue with the other activities
[aligned.webm](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/134537383/73b41fc6-261c-4d7a-8627-8f4c8a071f5c)